resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -584,6 +595,128 @@ class TextCursor
     }
 }
 
+[DSCResource()]
+class ColorFilter
+{
+    # Key required. Do not set.
+    [DscProperty(Key)] [string] $SID
+    [DscProperty()] [nullable[bool]] $FilterStatus
+    [DscProperty()] [ColorFilters] $FilterColor
+    [DscProperty()] [nullable[bool]] $KeyboardShortcutStatus
+
+    static hidden [string] $FilterStatusProperty = 'Configuration'
+    static hidden [string] $FilterStatusActiveProperty = 'Active'
+    static hidden [string] $FilterStatusValue = 'colorfiltering'
+    static hidden [string] $FilterColorProperty = 'FilterType'
+    static hidden [string] $KeyboardShortcutStatusProperty = 'HotkeyEnabled'
+
+    static [bool] GetStatus()
+    {
+        $FilterStatusArgs = @{  Path = $global:NTAccessibilityRegistryPath; Name = ([ColorFilter]::FilterStatusProperty)}

```

Comment:
```suggestion
        $FilterStatusArgs = @{  Path = $global:NTAccessibilityRegistryPath; Name = ([ColorFilter]::FilterStatusProperty)}

```

Is this the wrong registry path?

Reply:
No - as I explained in Teams, more than 1 Accessibility setting writes its name to this location. What needs to change here is to append instead of writing, as this will clobber the existing string data. And likewise to filter out instead of returning an empty string, to again avoid clobbering.